_posts/2019-07-29-vim-basics.markdown

```diff
@@ -0,0 +1,111 @@
+---
+layout: post
+title: vim Basics
+date: 2019-07-29 22:28:02
+categories:
+author: bronzdoc
+---
+
+# VIM basics
+
+I would not bore you by telling you the history of vim, and the differences between vi and vim and all of its philosophy...
+s
+If you are interested in that, you can give it a look [here](https://en.wikipedia.org/wiki/Vim_(text_editor)).
+
+<!--more-->
+
+So all you need to know is that vim is a text editor. In my opinion one of vim's strengths comes with how flexible it can be. It can be as simple or complex as you wish. It can be used to edit a simple config file on a server or you can make it a complete code editing environment.
+
+In this blog post I'll show you the basics of vim and provide you with the smallest amount of knowledge necessary to make vim useful for yourself, and yes I'll teach you how to exit ;)
+
+I'll assume you already have vim installed, if you are on a \*nix system you probably already have it.
+
+Open terminal and type: `vim vim-basic`.
+
+You'll see something like the following:
+```
+                                              VIM - Vi IMproved
+
+                                               version 8.1.500
+                                           by Bram Moolenaar et al.
+                                 Vim is open source and freely distributable
+
+                                        Help poor children in Uganda!
+                                type  :help iccf<Enter>       for information
+
+                                type  :q<Enter>               to exit
+                                type  :help<Enter>  or  <F1>  for on-line help
+                                type  :help version8<Enter>   for version info
+
+                                        Running in Vi compatible mode
+                                type  :set nocp<Enter>        for Vim defaults
+                                type  :help cp-default<Enter> for info on this
+```
```

Comment:
@bronzdoc I don't see this when I type that command. I see a blank new file

Reply:
You are correct @lubc 

I'm so used to type `vim` and then operate from there that i forgot that `vim filename` would not show that text. 🤦‍♂